Mopups/Mopups.Maui/Platforms/Windows/Impl/PopupPlatformWindows.cs

```diff
@@ -0,0 +1,101 @@
+﻿
+using Microsoft.Maui.Platform;
+using Mopups.Interfaces;
+using Mopups.Pages;
+using Mopups.Platforms.Windows;
+using Mopups.Services;
+
+namespace Mopups.Windows.Implementation
+{
+    class PopupPlatformWindows : IPopupPlatform
+    {
+        private IPopupNavigation PopupNavigationInstance => MopupService.Instance;
+
+        //public event EventHandler OnInitialized
+        //{
+        //    add => Popup.OnInitialized += value;
+        //    remove => Popup.OnInitialized -= value;
+        //}
+
+        //public bool IsInitialized => Popup.IsInitialized;
+
+        public bool IsSystemAnimationEnabled => true;
+
+        public PopupPlatformWindows()
+        {
+            //SystemNavigationManager.GetForCurrentView().BackRequested += OnBackRequested;
+        }
+
+        // Probably wire this through the life cycle parats like you have with android
+        //private async void OnBackRequested(object sender, BackRequestedEventArgs e)
+        //{
+        //    var lastPopupPage = PopupNavigationInstance.PopupStack.LastOrDefault();
+
+        //    if (lastPopupPage != null)
+        //    {
+        //        var isPrevent = lastPopupPage.DisappearingTransactionTask != null || lastPopupPage.SendBackButtonPressed();
+
+        //        if (!isPrevent)
+        //        {
+        //            e.Handled = true;
+        //            await PopupNavigationInstance.PopAsync();
+        //        }
+        //    }
+        //}
+
+        public async Task AddAsync(PopupPage page)
+        {
+            page.Parent = Application.Current.MainPage;
+
+            var popup = new global::Microsoft.UI.Xaml.Controls.Primitives.Popup();
+
+            // Use TOPLATFORM to create your handlers
+            // I'd recommend wiring up all your services through ConfigureMopups
+            // builder.Services.AddScoped<IPopupPlatform, PopupPlatform>();
+            // builder.Services.AddScoped<IPopupNavigation, PopupNavigation>();
+            // Then you can use contructor resolution instead of singletons
+            // But I figured we could do that in a later PR and just work on windows here
+
+            var renderer = (PopupPageRenderer)page.ToPlatform(Application.Current.MainPage.Handler.MauiContext);
```

Comment:
I noticed in a lot of the code it was instantiating the Handler by hand. 

If you use `ToPlatform` that sets everything up as intended. 

Also, I think eventually wiring everything up through "Services" will simplify a lot of the timing and remove the need for singletons. 

But that can be a later PR